### PR TITLE
Word snippet clean up - folders and descriptions

### PR DESCRIPTION
**Pull request by AlexJerabek**
Another pass on the Word snippets. This aims to standardize the names and descriptions. It also recategorizes the sample folders to give users a better idea of the content within.

### CODE REVIEW

File: samples/word/01-basics/basic-api-call.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 order: 1
 id: word-basics-basic-api-call
 name: Basic API call
```
Comment (kbrandl) [inline samples/word/01-basics/basic-api-call.yaml]: For parity with the name of the "Basic API call (JavaScript)", should this `name` attribute be set to "Basic API call (TypeScript)" ?

File: samples/word/02-paragraphs/insert-formatted-text.yaml [hunk 2]
```diff
@@ -1,4 +1,5 @@
-id: word-basics-insert-formatted-text
+order: 1
+id: word-paragraphs-insert-formatted-text
 name: Insert formatted text
 description: Shows how to use formatting objects as well as applying pre-built styles.
```
Comment (kbrandl) [inline samples/word/02-paragraphs/insert-formatted-text.yaml]: We've edited other snippet descriptions to remove the phrase "Shows how to..." -- should the same thing be done for this description?

File: samples/word/02-paragraphs/insert-in-different-locations.yaml [hunk 3]
```diff
@@ -1,6 +1,7 @@
-id: word-basics-insert-in-different-locations
-name: Insert locations.
-description: Shows how to insert content using the different available locations.
+order: 2
+id: word-paragraphs-insert-in-different-locations
+name: Insert at different locations
```
Comment (kbrandl) [inline samples/word/02-paragraphs/insert-in-different-locations.yaml]: Suggest changing `name` attribute value to:  **Insert content at different locations**

File: samples/word/03-images/insert-and-get-pictures.yaml [hunk 4]
```diff
@@ -1,6 +1,7 @@
-id: word-basics-insert-and-get-pictures
-name: Inline pictures
-description: Shows how to insert and get inline pictures.
+order: 1
+id: word-images-insert-and-get-pictures
+name: Using inline pictures
```
Comment (kbrandl) [inline samples/word/03-images/insert-and-get-pictures.yaml]: Consider changing "Using" to "Use" (for consistency with how other snippet names start out)

File: samples/word/04-ranges/split-words-of-first-paragraph.yaml [hunk 5]
```diff
@@ -1,6 +1,7 @@
-id: word-range-split-words-of-first-paragraph
-name: Range manipulation
-description: 'Show how to split a paragraph into word ranges and then traverses all the ranges to format each word, producing a "karaoke" effect.'
+order: 2
+id: word-ranges-split-words-of-first-paragraph
+name: Splitting a paragraph into ranges
```
Comment (kbrandl) [inline samples/word/04-ranges/split-words-of-first-paragraph.yaml]: Consider changing "Splitting" to "Split" in snippet name (for consistency with how other snippet names start out)

File: samples/word/05-tables/table-cell-access.yaml [hunk 6]
```diff
@@ -1,6 +1,7 @@
+order: 1
 id: word-tables-table-cell-access
-name: Table cell access
-description: Shows how to access a specific cell in a table.
+name: Creating and accessing a table
```
Comment (kbrandl) [inline samples/word/05-tables/table-cell-access.yaml]: Consider changing "Creating" to "Create" in snippet name (for consistency with how other snippet names start out)

File: samples/word/06-lists/insert-list.yaml [hunk 7]
```diff
@@ -1,3 +1,4 @@
+order: 1
 id: word-lists-insert-list
 name: List creation
```
Comment (kbrandl) [inline samples/word/06-lists/insert-list.yaml]: Consider changing "List creation" to "Create a list" in snippet name (for consistency with how other snippet names are phrased)

File: samples/word/50-common-patterns/multiple-property-set.yaml [hunk 8]
```diff
@@ -1,6 +1,7 @@
+order: 1
 id: word-common-patterns-multiple-property-set
-name: Multiple Property Set
-description: Setting multiple properties at once with the API object set() method.
+name: Multiple property set
```
Comment (kbrandl) [inline samples/word/50-common-patterns/multiple-property-set.yaml]: Consider changing name to something more like "Set multiple properties"

**Comment by AlexJerabek** [comment]
@kbrandl Thank you for the review. I've updated accordingly.

**Pull request merged**